lib/github_changelog_generator.rb

```diff
@@ -34,10 +34,15 @@ def initialize
     def run
       log = @generator.compound_changelog
 
-      output_filename = @options[:output].to_s
-      File.open(output_filename, "wb") { |file| file.write(log) }
-      puts "Done!"
-      puts "Generated log placed in #{Dir.pwd}/#{output_filename}"
+      if @options[:stdout]
+        puts log
+        puts "Done!"
```

Comment:
In the case of outputting to standard out, perhaps we can allow omitting the `Done!` text?

Reply:
Removed